servatrice/src/servatrice_database_interface.cpp

```diff
@@ -505,38 +505,38 @@ ServerInfo_User Servatrice_DatabaseInterface::evalUserQueryResult(const QSqlQuer
     if (!country.isEmpty())
         result.set_country(country.toStdString());
 
+    const QString privlevel = query->value(4).toString();
+    if (!privlevel.isEmpty())
+        result.set_privlevel(privlevel.toStdString());
+
     if (complete) {
-        const QString genderStr = query->value(4).toString();
+        const QString genderStr = query->value(5).toString();
         if (genderStr == "m")
             result.set_gender(ServerInfo_User::Male);
         else if (genderStr == "f")
             result.set_gender(ServerInfo_User::Female);
 
-        const QString realName = query->value(5).toString();
+        const QString realName = query->value(6).toString();
         if (!realName.isEmpty())
             result.set_real_name(realName.toStdString());
 
-        const QByteArray avatarBmp = query->value(6).toByteArray();
+        const QByteArray avatarBmp = query->value(7).toByteArray();
         if (avatarBmp.size())
             result.set_avatar_bmp(avatarBmp.data(), avatarBmp.size());
 
-        const QDateTime regDate = query->value(7).toDateTime();
-        if(!regDate.toString(Qt::ISODate).isEmpty()) {
+        const QDateTime regDate = query->value(8).toDateTime();
+        if (!regDate.toString(Qt::ISODate).isEmpty()) {
             qint64 accountAgeInSeconds = regDate.secsTo(QDateTime::currentDateTime());
             result.set_accountage_secs(accountAgeInSeconds);
         }
 
-        const QString email = query->value(8).toString();
+        const QString email = query->value(9).toString();
         if (!email.isEmpty())
             result.set_email(email.toStdString());
 
-        const QString clientid = query->value(9).toString();
+        const QString clientid = query->value(10).toString();
```

Comment:
If there's a way in the future to get value from the column name of the query result we wouldn't have to keep bumping numbers like this. Like if you could `query->value("clientId")`. This might already be possible, I haven't checked the docs.

No change needed for now, maybe something to look at later